src/phoenix/experimental/evals/templates/template.py

```diff
@@ -81,3 +81,27 @@ def _parse_variables(self) -> List[str]:
                 variables.append(variable_name)
 
         return variables
+
+
+def normalize_template(template: Union[PromptTemplate, str]) -> "PromptTemplate":
```

Comment:
centralized validation and normalization for re-use.

Reply:
Looks good for now. We've talked about simplifying the signature of `llm_eval_binary` to just accept an argument of type `PromptTemplate`, in which case this function would no longer be necessary.

Reply:
Makes sense